sdk/template/Azure.Template/samples/README.md

```diff
@@ -11,4 +11,13 @@ description: Samples for the Azure.Template client library.
 
 # Azure.Template Samples
 
-- [Creating Resource](https://github.com/Azure/azure-sdk-for-net/blob/3ac301ac6435c818ad7a9946ab1c4023cee236ff/eng/templates/Azure.Template/samples/Sample1_CreateResource.md)
\ No newline at end of file
+These code samples show common scenario operations with the Azure Template client library.
+
+|**Sample Name**|**Description**|
+|----------------|-------------|
+|[Sample1_HelloWorld][sample_helloworld] |Get secrets synchronously.|
```

Comment:
Tip: if you name the links the same token value you can use an abbreviated form:

```markdown
[Sample1_HelloWorld]
[Sample1_HelloWorldAsync]

[Sample1_HelloWorld]: link1
[Sample1_HelloWorldAsync]: link2
```